index.html

```diff
@@ -289,6 +289,15 @@ <h2>About me</h2>
           </ul>
         </div>
       </div>
+      <div class="contact">
+        <div class="contact-header">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
+        <div class="form">
+          <input type="text" name="name" class="input" placeholder="Full name">
```

Comment:
- [ ] Good job setting your inputs to their correct types. Let's improve our validation by adding the `maxlength="30"` attribute to our name input, this way our users won't be able to over-extend and maybe type something else that isn't our name.